src/raft.rs

```diff
@@ -467,10 +467,10 @@ impl<T: Storage> Raft<T> {
             "msg" => ?m,
         );
         m.from = self.id;
-        if m.msg_type() == MessageType::MsgRequestVote
-            || m.msg_type() == MessageType::MsgRequestPreVote
-            || m.msg_type() == MessageType::MsgRequestVoteResponse
-            || m.msg_type() == MessageType::MsgRequestPreVoteResponse
+        if m.get_msg_type() == MessageType::MsgRequestVote
```

Comment:
This (and many other getters) violates https://github.com/rust-lang/rfcs/blob/master/text/0344-conventions-galore.md#gettersetter-apis, see https://github.com/pingcap/raft-rs/issues/223

Reply:
I'm afraid the original method is not safe to be used. prost will drop unknown enums silently with `value()`. `get_value` makes sure unknown enums will lead to panic, which is not the best way but ensures data consistency.